caffe2/operators/given_tensor_fill_op.cc

```diff
@@ -26,32 +26,137 @@ OPERATOR_SCHEMA(GivenTensorFill)
     .NumInputs(0, 1)
     .NumOutputs(1)
     .AllowInplace({{0, 0}})
+    .Arg(
+        "values",
+        "The value for the elements of the output tensor.",
```

Comment:
Nit: add the type (e.g., list of ints or floats) in the description for each argument?